ci-operator/jobs/infra-periodics.yaml

```diff
@@ -1195,3 +1195,55 @@ periodics:
     - name: token
       secret:
         secretName: github-credentials-openshift-bot
+- agent: kubernetes
+  cluster: app.ci
+  decorate: true
+  extra_refs:
+  - base_ref: master
+    org: openshift
+    repo: release
+  hidden: true
+  interval: 12h
+  labels:
+    ci.openshift.io/role: infra
+  name: periodic-ci-secret-bootstrap
+  spec:
+    containers:
+    - args:
+      - --bw-password-path=/etc/ci-secret-bootstrap/bw_password
+      - --bw-user=openshift-dev-infra+ci-bitwarden-bot@redhat.com
+      - --config=./core-services/ci-secret-bootstrap/_config.yaml
+      - --dry-run=false
+      - --force=false
+      command:
+      - /usr/bin/ci-secret-bootstrap
+      env:
+      - name: KUBECONFIG
+        value: /etc/build-farm-credentials/sa.config-updater.api.ci.config:/etc/build-farm-credentials/sa.config-updater.app.ci.config:/etc/build-farm-credentials/sa.config-updater.build01.config
+      image: registry.svc.ci.openshift.org/ci/ci-secret-bootstrap:latest
+      imagePullPolicy: Always
+      name: ci-secret-bootstrap
+      resources:
+        requests:
+          cpu: 500m
```

Comment:
Some ppl mine bitcoins, we read data from Bitwarden.

Reply:
Copied from other jobs.
I got your point tho.